tests/apps/qchem/test_vibronic.py

```diff
@@ -232,3 +232,45 @@ def test_sample_integration(p, integration_sample_number):
     assert dims == (integration_sample_number, len(alpha) * 2)
     assert samples.dtype == "int"
     assert (samples >= 0).all()
+
+
+class TestOperation:
+    """Tests for the function ``strawberryfields.apps.qchem.vibronic.VibronicTransition``"""
+
+    U1 = np.array([[-0.17905859, 0.98383841], [0.98383841, 0.17905859]])
+
+    r = np.array([0.00387885, -0.20415111])
+
+    U2 = np.array([[-0.33417234, 0.94251199], [0.94251199, 0.33417234]])
+
+    alpha = np.array([0.38558533, 2.99728422])
+
+    prob = np.array(
+        [
+            [1.49817854e-04, 1.27328834e-03, 5.43505231e-03, 1.55353201e-02],
+            [2.26762512e-06, 2.68454082e-05, 1.52080070e-04, 5.56423985e-04],
+            [2.80701727e-06, 2.52203270e-05, 1.14702166e-04, 3.51760396e-04],
+            [1.14518212e-07, 1.37859685e-06, 7.96673530e-06, 2.98259165e-05],
+        ]
+    )
+
+    p1 = [U1, r, U2, alpha, prob]
+
+    @pytest.mark.parametrize("params", [p1])
+    def test_operation(self, params):
+        """Test if the function ``strawberryfields.apps.qchem.vibronic.VibronicTransition`` gives
+        the correct probabilities of all possible Fock basis states when used in a circuit"""
+
+        U1, r, U2, alpha, prob = params
+
+        eng = sf.LocalEngine(backend="gaussian")
+
+        gbs = sf.Program(len(U1))
+
+        with gbs.context as q:
+
+            vibronic.VibronicTransition(U1, r, U2, alpha) | q
```

Comment:
Any negatives of this approach? I can't think of any.

Reply:
Seems to be working fine.

Reply:
I really think this is best. In practice, I would prefer to use 
``` pythn3
from strawberryfields.qchem.vibronic import VibtronicTransition
```
but we don't need to do that in a test